src/main/kotlin/ink/abb/pogo/scraper/gui/SocketServer.kt

```diff
@@ -0,0 +1,218 @@
+package ink.abb.pogo.scraper.gui
+
+import POGOProtos.Data.PokemonDataOuterClass
+import com.corundumstudio.socketio.Configuration
+import com.corundumstudio.socketio.SocketIOServer
+import com.pokegoapi.api.map.fort.Pokestop
+import com.pokegoapi.api.player.PlayerProfile
+import com.pokegoapi.api.pokemon.Pokemon
+import com.pokegoapi.google.common.geometry.S2LatLng
+import ink.abb.pogo.scraper.Context
+import ink.abb.pogo.scraper.util.pokemon.getIvPercentage
+
+class SocketServer {
+    val requiredXp = arrayOf(0, 1000, 3000, 6000, 10000, 15000, 21000, 28000, 36000, 45000, 55000, 65000, 75000,
```

Comment:
Now we have the same magic number array twice in the code, maybe refactor so it's only defined once